typings/internal-electron.d.ts

```diff
@@ -138,12 +138,16 @@ declare namespace Electron {
     acceleratorWorksWhenHidden?: boolean;
   }
 
-  interface IpcMainEvent {
+  interface ReplySender {
     sendReply(value: any): void;
   }
 
+  interface IpcMainEvent {
+    _replySender: ReplySender;
+  }
+
   interface IpcMainInvokeEvent {
-    sendReply(value: any): void;
+    _replySender: ReplySender;
     _reply(value: any): void;
```

Comment:
Ok this is confusing, what is `_replySender.sendReply` vs `_reply` 🤔 We should rename one of these while we're doing this

Reply:
i deleted `_reply`.